lib/rules/no-unauthorized-global-properties.js

```diff
@@ -0,0 +1,57 @@
+"use strict";
+
+var defaultAllowedProperties = [
+  "addEventListener",
+  "clearInterval",
```

Comment:
Technically anything is global that's on the `window` object in the browser.

So all the properties and methods here: https://developer.mozilla.org/en-US/docs/Web/API/Window